### DISCUSSION TITLE
Show your Rspress site!

### DISCUSSION THREAD

**Discussion by sanyuan0704** [web-infra-dev/rspress#607] [Show and tell]
Welcome to add the website built by Rspress.

Just post a comment below in this GitHub discussion with some basic information:

- **Site title**: <= 60 characters 
- **Site description**: ≤120 characters 
- **Site URL**: live production deployment URL of your site 
- **Site source code URL**: only for open-source sites, you can add the GitHub repo url.

Example: 

- **Site title**: my blog 
- **Site description**: a blog site including all my articles based on custom blog theme
- **Site URL**: https://evanmiao.com/
- **Site source code URL**: https://github.com/evanmiao

We will receive the site information and then display it in Rspress official website. Expect your post!

**Comment by chenjiahan** [comment]
- Site title: Rsbuild
- Site description: The Rspack-based Build Tool
- Site URL: https://rsbuild.dev/
- Site source code URL: https://github.com/web-infra-dev/rsbuild

![image](https://github.com/web-infra-dev/rspress/assets/7237365/05093024-1eb6-425f-b4dc-46c061ab22fe)

**Reply by 10Derozan** [reply]
You two integrate with each other, hh

**Comment by Yukiniro** [comment]
- Site title: toukey.js
- Site description: A simple and efficient keyboard events library. 
- Site URL: https://toukey.vercel.app/
- Site source code URL: https://github.com/Yukiniro/toukey

**Comment by chenjiahan** [comment]
- Site title: Re.Pack
- Site description: A toolkit to build your React Native application with Webpack
- Site URL: https://re-pack.dev/
- Site source code URL: https://github.com/callstack/repack

![image](https://github.com/web-infra-dev/rspress/assets/7237365/f04e7ed5-40de-4e17-ae1c-e22fce694cdb)

**Comment by chenjiahan** [comment]
- Site title: MarsCode
- Site description: MarsCode is an AI-powered cloud-based IDE.
- Site URL: https://docs.marscode.com/

![image](https://github.com/web-infra-dev/rspress/assets/7237365/a6d530f3-701d-4397-9b63-2f4f98b30231)

**Comment by zhoushaw** [comment]
* Site title: Module Federation
* Site description: Module Federation is a concept that allows developers to share code and resources across multiple JavaScript applications.
* Site URL: https://module-federation.io/

![image](https://github.com/web-infra-dev/rspress/assets/27547179/dfbcd160-f863-4816-92f6-d09a34e943f9)

**Comment by chenjiahan** [comment]
- Site title: React Native Testing Library
- Site description: Helps you to write better tests with less effort.
- Site URL: https://callstack.github.io/react-native-testing-library/
- Site source code URL: https://github.com/callstack/react-native-testing-library

![image](https://github.com/web-infra-dev/rspress/assets/7237365/05c390a1-63a1-40ec-836f-e6804fba1d79)

**Comment by chenjiahan** [comment]
- Site title: svelte-markdoc-preprocess
- Site description: Bring the power of Markdoc right into your Svelte applications.
- Site URL: https://svelte-markdoc-preprocess.pages.dev/
- Site source code URL: https://github.com/TorstenDittmann/svelte-markdoc-preprocess

![image](https://github.com/web-infra-dev/rspress/assets/7237365/2e6f2bfb-bace-4c9a-b573-4ef864e8b76f)

**Comment by asurance** [comment]
Site title: openify
Site description: A tool that facilitates the use of React pop-up components
Site URL: https://asurance.github.io/openify/
Site source code URL: https://github.com/asurance/openify

**Comment by BytesZero** [comment]
- Site title: FlutterAds
- Site description: FlutterAds 致力于构建优质的 Flutter 广告插件
- Site URL: https://flutterads.top/
- Site source code URL: https://github.com/FlutterAds/site

![image](https://github.com/user-attachments/assets/8d805e90-b97a-4bae-b67c-1f126a2d12a6)

**Comment by jbroma** [comment]
- Site title: React Native Bottom Tabs
- Site description: Bottom Tabs using native platform primitives.
- Site URL: [okwasniewski.github.io/react-native-bottom-tabs/](https://okwasniewski.github.io/react-native-bottom-tabs/)
- Site source code URL: https://github.com/okwasniewski/react-native-bottom-tabs/

<img width="1509" alt="image" src="https://github.com/user-attachments/assets/81497b86-ade8-4242-8f0d-1cfe3806f0bf">

**Comment by quanru** [comment]
Site title: LifeOS for Obsidian
Site description: Life OS Management system based on Obsidian(PARA Method/Periodic Notes/Calendar)
Site URL: https://lifeos.vip

![image](https://github.com/user-attachments/assets/52039c3c-75fc-4c26-9df9-36cc353e8f8e)

**Comment by nordowl** [comment]
**Site title:** Filenest
**Site description:** Docs for a React library to integrate your external file hoster directly into your app with custom UI.
**Site URL:** https://filenest.github.io/filenest/
**Site source code URL:** https://github.com/filenest/filenest

![grafik](https://github.com/user-attachments/assets/165a1f35-6f57-4c1d-8246-759132b99a3e)

**Comment by chenjiahan** [comment]
- Site title: Lynx
- Site description: a family of technologies empowering developers to use their existing web skills to create truly native UIs for both mobile and web from a single codebase.
- Site URL: https://lynxjs.org/
- Site source code URL: https://github.com/lynx-family/lynx-website

![download](https://github.com/user-attachments/assets/88e70fd5-e898-4169-af05-c7e59331703c)

**Comment by Timeless0911** [comment]
Site title: FlowGram.AI
Site description: a node-based flow building engine that helps developers quickly create workflows in either fixed layout or free connection layout modes, providing a set of interactive best practices. It's particularly suitable for visual workflows with clear inputs and outputs.
Site URL: [flowgram.ai](https://flowgram.ai/en)
Site source code URL: [bytedance/flowgram.ai](https://github.com/bytedance/flowgram.ai/tree/main/apps/docs)

![image](https://github.com/user-attachments/assets/28d52422-31d6-4854-88b1-ff4f9bcd5ef8)

**Comment by Timeless0911** [comment]
Site title: AIScript
Site description: a unique combination of interpreter programming language and web framework, both written in Rust, designed to help developers build AI applications effortlessly. 
Site URL: [AIScript](https://aiscript.dev)
Site source code URL: [aiscriptdev/aiscript-docs](https://github.com/aiscriptdev/aiscript-docs)

![image](https://github.com/user-attachments/assets/e99025c2-af6e-48fe-9b70-e3747b772ba6)

**Reply by Timeless0911** [reply]
@folyd Awesome work for this project and thank you for your support!

**Comment by Huxpro** [comment]
Site title: Lynx 
Site description: Official website of the Lynx family of technologies
Site URL: https://lynxjs.org
Site source code URL: https://github.com/lynx-family/lynx-website

![image](https://github.com/user-attachments/assets/3c632433-ce3b-48f8-9ce1-8a41d6c3f6de)

**Comment by JounQin** [comment]
- Site title: Alauda Container Platform
- Site description: Accelerate Innovation with Alauda: The AI-Ready Cloud Native Solution
- Site URL: https://docs.alauda.io
- Site source code URL: N/A for now, it will be open-sourced in a few months

Our show case is a bit special because we have 19 sub sites x N versions all in one! 🤯

<img width="2200" alt="iShot_2025-04-30_21 27 39" src="https://github.com/user-attachments/assets/950b9a02-47b6-4d53-a67f-08f56392cc83" />

**Comment by atticusofsparta** [comment]
Site title: Harlequin Docs
Site description: Documentation site for the Harlequin Toolkit ecosystem of products
Site URL: https://docs_harlequin.arweave.net
Site source code URL: https://github.com/the-permaweb-harlequin/harlequin-toolkit/tree/main/docs

<img width="2878" height="1812" alt="image" src="https://github.com/user-attachments/assets/910e822d-4f5c-466d-8333-150530514697" />

**Comment by Karibash** [comment]
Site title: byethrow
Site description: A lightweight Result type library
Site URL: https://praha-inc.github.io/byethrow/
Site source code URL: https://github.com/praha-inc/byethrow/tree/main/website

<img width="3600" height="2576" alt="praha-inc github io_byethrow_ (1)" src="https://github.com/user-attachments/assets/9dd6b30d-eb71-46fb-8363-33683a577364" />

**Comment by SoonIter** [comment]
- Site title: Xiaomi MiMo
- Site description: Xiaomi's universal smart platform for advanced AI models and human-machine collaboration.
- Site URL: https://mimo.xiaomi.com/
<img width="1603" height="998" alt="image" src="https://github.com/user-attachments/assets/45a7b010-3c4c-4d83-ab05-60056dbdc51b" />

**Comment by ScottMorse** [comment]
- **Site title**: pacwich
- **Site description**: Documentation for `pacwich`: Lightweight monorepo tooling on top of Bun, npm, or pnpm workspaces
- **Site URL**: https://pacwich.dev
- **Site source code URL**: https://github.com/smorsic/pacwich (at workspaces/web/documentation-website)

<img width="2417" height="1882" alt="screenshot_000005" src="https://github.com/user-attachments/assets/6e449178-70e0-4c2e-a3ba-31e2cd6bdb57" />

**Comment by SoonIter** [comment]
- **Site title**: React Native Paper Dates
- **Site description**: Material Design 3 date and time pickers for React Native with interactive docs.
- **Site URL**: https://web-ridge.github.io/react-native-paper-dates/
- **Site source code URL**: https://github.com/web-ridge/react-native-paper-dates

<img width="1616" height="1010" alt="image" src="https://github.com/user-attachments/assets/9fc3b0a3-94ec-4c79-a4df-a27ef22de6df" />



Migration PR: https://github.com/web-ridge/react-native-paper-dates/pull/531

**Comment by SoonIter** [comment]
- **Site title**: Taroify
- **Site description**: A lightweight Taro React UI component library with custom docs theme and component simulator.
- **Site URL**: https://taroify.github.io/taroify.com/introduce/index.html
- **Site source code URL**: https://github.com/mallfoundry/taroify

<img width="828" height="559" alt="image" src="https://github.com/user-attachments/assets/187bde1b-2ba6-4d16-aa01-976c7107f816" />


Migration PR: https://github.com/mallfoundry/taroify/pull/1020

**Comment by sanjaiyan-dev** [comment]
**Site title:** rspress-plugin-third-parties

**Site description:** High-performance third-party embeds, GTM, Google Analytics, and non-blocking script strategies for Rspress sites.

**Site URL:** https://sanjaiyan-dev.github.io/rspress-plugin-third-parties

**Site source code URL:** https://github.com/sanjaiyan-dev/rspress-plugin-third-parties

<img width="1920" height="1536" alt="120shots_so" src="https://github.com/user-attachments/assets/1bb336b8-cb7c-4421-a672-3808e8c976d3" />

**Comment by gxy5202** [comment]
- Site title: Addfox
- Site description: Accelerate your browser extension development. Built on Rsbuild — fast, simple, and free.
- Site URL: https://addfox.dev
- Site source code URL: https://github.com/addfox/addfox
<img width="2536" height="1305" alt="image" src="https://github.com/user-attachments/assets/b8308033-31d3-4f17-9849-68d6fc02f630" />

**Comment by Karibash** [comment]
- Site title: `@praha/toi`
- Site description: A tiny, headless React utility for imperative dialogs and toasts.
- Site URL: https://praha-inc.github.io/toi/
- Site source code URL: https://github.com/praha-inc/toi/tree/main/website

<img width="3600" height="1840" alt="test" src="https://github.com/user-attachments/assets/a7cfef70-f4ba-48da-b464-5f03d6363327" />

**Comment by sanjaiyan-dev** [comment]
- **Site title:** URAI (உரை) – AST-Aware Codebase-to-Prompt Compiler
- **Site description:** Rust SWC compiler reducing JS/TS prompt tokens by up to 80% via AST pruning, Tailwind stripping, and Ollama summaries.
- **Site URL:** https://sanjaiyan-dev.github.io/urai-ecma/
- **Site source code URL:** https://github.com/sanjaiyan-dev/urai-ecma

<img width="1920" height="1440" src="https://github.com/user-attachments/assets/7c1179bd-a3ce-4235-9402-df599009fccf" alt="Mobile mockups displaying the responsive dark-mode documentation of URAI (urai-ecma), showcasing the AST-aware prompt engine homepage and interactive Rust compiler HUD"  />
